### ISSUE TITLE
Update R code for generating charts

### ISSUE THREAD

**Issue by mfebrizio** [regulatorystudies/Reg-Stats#8]
Main tasks:

- update R code for each chart, including combining code between .Rmd and .ipynb files as needed
- keep only one file for each chart (retain helper files, such as style.R, local_utils.R, refresh_data.R, demos, etc.)
- add cross-purpose code to: 
  - style.R -- if related to style/theme
  - local_utils.R -- if related to general functionality (e.g., get_png function)
- remove cross-purpose code from individual chart files
  - in other words, rather that defining get_png in each file, define it once in local_utils.R, then use source() to import the function for use in each file

**Comment by mfebrizio** [comment]
Completed or covered more specifically by other issues.
